src/common/XChainIntegrator.sol

```diff
@@ -109,7 +120,6 @@ abstract contract XChainIntegrator is Auth {
 	) public onlyRole(MANAGER) {
```

Comment:
I think we need to verify that we're allowed to send to kens to the `destinationAddress` right?